CondFormats/BTauObjects/src/BTagEntry.cc

```diff
@@ -17,8 +22,8 @@ BTagEntry::Parameters::Parameters(
   float discr_max
 ):
   operatingPoint(op),
-  measurementType(measurement_type),
-  sysType(sys_type),
+  measurementType(makeLowerCase(measurement_type)),
+  sysType(makeLowerCase(sys_type)),
```

Comment:
Please remove the two unneeded copies in `std::string makeLowerCase(std::string data)` function, and make it `static`:
````
static void makeLowerCase(std::string& data) {
  std::transform(data.begin(), data.end(), data.begin(), ::tolower);
}
````